app/qml/editor/RelationPhotoDelegate.qml

```diff
@@ -0,0 +1,84 @@
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+import QtQuick 2.14
+import QtQuick.Controls 2.14
+import QtQml.Models 2.14
+import QtQuick.Layouts 1.14
+import QtGraphicalEffects 1.14
+
+import QgsQuick 0.1 as QgsQuick
+import lc 1.0
+import ".."
+
+Item {
+  id: root
+
+  signal clicked( var feature )
+
+  height: parent.height
+  width: height
+
+  Image {
+    id: image
+
+    property bool imageValid: true
+
+    anchors.centerIn: parent
+    width: imageValid ? parent.width : parent.width * 0.4
+    height: imageValid ? parent.width : parent.width * 0.4
+    sourceSize.width: image.width
+    sourceSize.height: image.height
+    visible: imageValid
+
+    autoTransform: true
+    source: {
+      let absolutePath = model.PhotoPath
+
+      if (image.status === Image.Error) {
+        image.imageValid = false
+        customStyle.icons.notAvailable
+      }
+      else if (absolutePath !== '' && __inputUtils.fileExists(absolutePath)) {
+        "file://" + absolutePath
+      }
+      else if (absolutePath === '' || absolutePath === undefined) {
+        image.imageValid = false
+        customStyle.icons.notAvailable
```

Comment:
would fall too else branch, no?

Reply:
Else branch did the same as this one so I removed this redundant check